personal_assistant/menus.py

```diff
@@ -86,62 +85,62 @@ def contacts_menu():
         elif command == "help":
             print(messages.contacts_help_center)
 
-        elif command == "add":
-            print(add_contact(args))
+        elif command in ("add-contact", "add-note"):
```

Comment:
are you sure "add-note" is a proper command to add a contact?

Reply:
yes, as tells the contacts interface to create smth new